src/main/python/systemml/converters.py

```diff
@@ -212,10 +215,38 @@ def convertToNumPyArr(sc, mb):
     else:
         raise TypeError('sc needs to be of type SparkContext') # TODO: We can generalize this by creating py4j gateway ourselves
 
+# Returns the mean of a model if defined otherwise None
+def getModelMean(model_name):
+    """
+    Input Parameters
+    ----------------
+    model_name: Name of the model
+
+    Returns
+    -------
+    mean: Mean value of model if its defined in the list MODEL_MEAN else None.
+
+    """
+
+    try:
+        mean = MODEL_MEAN[model_name.upper()]
+    except:
+        mean = None
+    return mean
+
+
 # Example usage: convertImageToNumPyArr(im, img_shape=(3, 224, 224), add_rotated_images=True, add_mirrored_images=True)
 # The above call returns a numpy array of shape (6, 50176) in NCHW format
-def convertImageToNumPyArr(im, img_shape=None, add_rotated_images=False, add_mirrored_images=False):
-    from PIL import Image
+def convertImageToNumPyArr(image_file_name, img_shape=None, add_rotated_images=False, add_mirrored_images=False,
```

Comment:
@asurve  I would not recommend changing image object to image file.  Consider this example: http://apache.github.io/systemml/beginners-guide-caffe2dml#how-to-prepare-a-directory-of-jpeg-images-for-training-with-caffe2dml-